boards/shields/rtkmipilcdb00000be/Kconfig.defconfig

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_RTKMIPILCDB00000BE
+if DISPLAY
+
+config INPUT_INIT_PRIORITY
```

Comment:
Please add a comment as to why the input priority needs to be changed

Reply:
Done